### ISSUE TITLE
Color selector available on Funnel visualizations

### ISSUE THREAD

**Issue by blackush** [metabase/metabase#14962]
Hi Guys,
any chance you can add to the Funnel chart an option for color selection as all the other charts ? super critical for us :)

**Comment by ab-smith** [comment]
crazy that there is not that much of  thumbs up and too bad that I'm limited to one :)

**Comment by arkahgdm** [comment]
Yes please add some color options to the funnel, it would help us customize better looking dashboards! Love the visualization option otherwise.

**Comment by vitorleal** [comment]
Also looking for this
